Source/Core/Core/CoreTiming.cpp

```diff
@@ -322,10 +324,15 @@ void CoreTimingManager::Advance()
     std::pop_heap(m_event_queue.begin(), m_event_queue.end(), std::greater<Event>());
     m_event_queue.pop_back();
 
+    LWDProfiler::GetCurrentProfiler().SetMarker("Throttle");
     Throttle(evt.time);
+
+    LWDProfiler::GetCurrentProfiler().SetMarker(*evt.type->name);
     evt.type->callback(system, evt.userdata, m_globals.global_timer - evt.time);
   }
 
+  LWDProfiler::GetCurrentProfiler().SetMarker("External Exceptions");
```

Comment:
Can we rename this Marker as "Scheduler" or "CoreTiming" and also set it right at the start of Advance so it covers the entire execution of Advance?